### PR TITLE
Add ActionController::Live::Buffer#writeln the write a line to the stream with a newline included

### PR DESCRIPTION
**Pull request by dhh**
```ruby
    send_stream(filename: "subscribers.csv") do |stream|
      stream.writeln "email_address,updated_at"
    
      @subscribers.find_each do |subscriber|
        stream.writeln [ subscriber.email_address, subscriber.updated_at ].join(",")
      end
    end
```

### CODE REVIEW

**Comment by kaspth** [comment]
Could also consider adding `print` and `puts` aliases to match Ruby's socket superclass `IO`.

File: actionpack/lib/action_controller/metal/live.rb [hunk 1]
```diff
@@ -163,6 +163,11 @@ def write(string)
         end
       end
 
+      # Same as +write+ but automatically include a newline at the end of the string.
+      def writeln(string)
+        write "#{string}\n"
```
Comment (kaspth) [inline actionpack/lib/action_controller/metal/live.rb:168]: Do we want to guard for the line already ending in a `\n`? This was just lifted from my comparison to `IO#puts`, which does that.

Comment (dhh) [reply actionpack/lib/action_controller/metal/live.rb:168]: Yeah I dig that.

**Comment by dhh** [comment]
Considered both of those names, but ultimately decided that they don't actually explain WHY that means adding a newline. Same with puts vs print. I mean, you learn that one does one thing, the other does the other. But I don't see a rhythm to it. Imo writeln is clearer.

**Review by rafaelfranca — APPROVED** [review]

**Comment by ioquatix** [comment]
`puts` comes from C's `stdio.h` and follows a convention established for like, probably the past 30+ years. I agree, `writeln` or even better `write_line` is clearer, but at least alias is advisable.

**Pull request merged**